tilelang/jit/adapter/wrapper.py

```
@@ -1230,6 +954,7 @@ def __init__(self, target: Target):
     def wrap(self, c_source: str):
         # assert self.scheduled_ir_module is not None, "Please assign optimized module first."
         if is_cuda_target(self.target):
+            from tilelang.jit.adapter.nvrtc import TLNVRTCSourceWrapper
             wrapper_class = TLNVRTCSourceWrapper
         else:
             raise ValueError(f"Unsupported platform: {self.arch.platform}")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)